### ISSUE TITLE
Fix unclosed `supplied` element at homerichymns.xml 4.568a

### ISSUE THREAD

**Issue by whoopsedesy** [sasansom/sedes#88]
Commit 99fea693629801ea07c588543ec5214bff117142 introduced a well-formedness error into homerichymns.xml. A left square bracket was changed to a `<supplied>` start tag (part of #85), but there is no `</supplied>` end tag to balance it.

https://github.com/sasansom/sedes/commit/99fea693629801ea07c588543ec5214bff117142#diff-4449804ff6e0fa14797e39d04e00dd21607f66206892feab87900e5854c536b9L1874-L1880

```diff
 <lb />tau=t' e)/xe, *maia/dos ui(e/, kai\ a)grau/lous e(/likas bou=s
 <lb />i(/ppous t' a)mfipo/leue kai\ h(mio/nous talaergou/s.</p></sp>
 <p>
-<lb rend="displayNum" n="568a" /> [w(s e)/fat': ou)rano/qen de\ path\r *zeu\s au)to\s e)/pessi
+<lb rend="displayNum" n="568a" /> <supplied reason="lost">w(s e)/fat': ou)rano/qen de\ path\r *zeu\s au)to\s e)/pessi
 <lb rend="displayNum" n="568b" /> qh=ke te/los: pa=sin d' a)/r' o(/ g' oi)wnoi=si ke/leusen
 <lb />kai\ xaropoi=si le/ousi kai\ a)rgio/dousi su/essi
 <lb rend="displayNum" n="570" />kai\ kusi\ kai\ mh/loisin, o(/sa tre/fei eu)rei=a xqw/n,
```

The well-formedness error is revealed by e.g. [Tidy](https://www.html-tidy.org/):

```
$ tidy -xml corpus/homerichymns.xml 
line 1888 column 54 - Error: unexpected </p> in <supplied>
line 1892 column 10 - Error: unexpected </div1> in <supplied>
line 2723 column 17 - Error: unexpected </body> in <supplied>
line 2723 column 24 - Error: unexpected </text> in <supplied>
line 2724 column 1 - Error: unexpected </TEI.2> in <supplied>
Tidy found 0 warnings and 5 errors!
```

Our [src/tei.py](https://github.com/sasansom/sedes/blob/8fe7296ab179d1e0e25de75bbce406cdc4ca7290/src/tei.py) does not report the error, but it causes the last line of book 4 two be written to CSV with `book_n=5` and without line numbers:

```diff
 Hom.Hymn,4,579,7,καὶ,καί,8,–,auto,1,"καὶ σὺ μὲν οὕτω χαῖρε, Διὸς καὶ Μαιάδος, υἱέ·"
 Hom.Hymn,4,579,8,μαιάδος,μαιάς,9,–⏑⏑,auto,1,"καὶ σὺ μὲν οὕτω χαῖρε, Διὸς καὶ Μαιάδος, υἱέ·"
 Hom.Hymn,4,579,9,υἱέ,υἱός,11,––,auto,1,"καὶ σὺ μὲν οὕτω χαῖρε, Διὸς καὶ Μαιάδος, υἱέ·"
-Hom.Hymn,4,580,1,αὐτὰρ,ἀτάρ,1,–⏑,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
-Hom.Hymn,4,580,2,ἐγὼ,ἐγώ,2.5,⏑–,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
-Hom.Hymn,4,580,3,καὶ,καί,4,–,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
-Hom.Hymn,4,580,4,σεῖο,σύ,5,–⏑,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
-Hom.Hymn,4,580,5,καὶ,καί,6.5,⏑,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
-Hom.Hymn,4,580,6,ἄλλης,ἄλλος,7,––,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
-Hom.Hymn,4,580,7,μνήσομ’,μιμνήσκω,9,–⏑,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
-Hom.Hymn,4,580,8,ἀοιδῆς,ἀοιδή,10.5,⏑––,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
+Hom.Hymn,5,,1,αὐτὰρ,ἀτάρ,1,–⏑,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
+Hom.Hymn,5,,2,ἐγὼ,ἐγώ,2.5,⏑–,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
+Hom.Hymn,5,,3,καὶ,καί,4,–,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
+Hom.Hymn,5,,4,σεῖο,σύ,5,–⏑,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
+Hom.Hymn,5,,5,καὶ,καί,6.5,⏑,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
+Hom.Hymn,5,,6,ἄλλης,ἄλλος,7,––,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
+Hom.Hymn,5,,7,μνήσομ’,μιμνήσκω,9,–⏑,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
+Hom.Hymn,5,,8,ἀοιδῆς,ἀοιδή,10.5,⏑––,auto,1,αὐτὰρ ἐγὼ καὶ σεῖο καὶ ἄλλης μνήσομ’ ἀοιδῆς.
 Hom.Hymn,5,1,1,μοῦσά,Μοῦσα,1,–⏑,auto,1,"μοῦσά μοι ἔννεπε ἔργα πολυχρύσου Ἀφροδίτης,"
 Hom.Hymn,5,1,2,μοι,ἐγώ,2.5,⏑,auto,1,"μοῦσά μοι ἔννεπε ἔργα πολυχρύσου Ἀφροδίτης,"
 Hom.Hymn,5,1,3,ἔννεπε,ἐνέπω,3,–⏑⏑,auto,1,"μοῦσά μοι ἔννεπε ἔργα πολυχρύσου Ἀφροδίτης,"
```

**Comment by whoopsedesy** [comment]
The Loeb edition says of [lines 568a and 568b](https://github.com/sasansom/sedes/blob/cf69dd879a7a990b9d47298dc2f8f121b999b495/corpus/homerichymns.xml#L1877-L1878) "these two lines have been supplemented by the editor Allen." @sasansom is accordingly going to add a `</supplied>` tag at the end of line 568b.

The [Perseus 5.0 version](https://github.com/PerseusDL/canonical-greekLit/blob/645148145584f04f4d2535e9c90e2f7373611908/data/tlg0013/tlg004/tlg0013.tlg004.perseus-grc2.xml#L700-L701) uses two `<supplied></supplied>` elements, one per line, but also strangely retains the square brackets.

**Comment by whoopsedesy** [comment]
Just checking--was d6435a798ae02bff8af4c4fba84c11d22689cc43 intended to extend over 569 as well, as it does?

**Comment by sasansom** [comment]
Yes indeed——I extended it over 569 since the two lines form a single syntactic unit.

**Comment by whoopsedesy** [comment]
But the way you've done it, it extends over *three* lines: 568a, 568b, 569. 569 isn't explicitly numbered but it's right after 568b.

**Comment by sasansom** [comment]
Ah, thanks for catching this. Fixed with commit [1ee0bc1](https://github.com/sasansom/sedes/commit/1ee0bc19c1d77303a73730e25ea8a095f049ee5b).
